src/object.d

```diff
@@ -3190,11 +3199,24 @@ void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
  * Create RTInfo for type T
  */
 
+template RTInfoImpl(size_t[] pointers)
+{
+    immutable size_t[pointers.length] data = pointers[];
+    immutable RTInfoImpl = data.ptr;
```

Comment:
The way it's done atm. means 1/64th more memory traffic which translates to 1/64th slower marking for big heaps. Should we at least allocate it lazily, there are typically big regions of NO_SCAN data, e.g. strings and number arrays.